### PR TITLE
Load config file with path

### PR DESCRIPTION
**Pull request by yongpingchen**
Refer #8 
### Changes
- specify path of config file through env var `GWM_CONFIG`
- location of db can be configured in config file with `db`

### CODE REVIEW

**Comment by satoshikumano** [comment]
Could you explain the background why DB path setting is useful?

**Comment by satoshikumano** [comment]
Why not use global option belongs to `App` ?

https://github.com/urfave/cli#flags

**Comment by satoshikumano** [comment]
It is bit strange that each command takes the configuration file path since it may not change so often.

**Review by satoshikumano — COMMENTED** [review]
Left comments.

**Comment by yongpingchen** [comment]
> Could you explain the background why DB path setting is useful?
Not required by customer. I think we also need to make it configurable. Base on the reasons: 
1. If binary file is located to the binary specified folder like `bin` folder, it may be better to make it possible to generate db file in other folder. 
2. developer may want to separate binary with file generated by execution. Like they can put config file and db file together.

**Comment by yongpingchen** [comment]
> Why not use global option belongs to App ?

Agreed.

**Comment by yongpingchen** [comment]
> It is bit strange that each command takes the configuration file path since it may not change so often.

We can make it configured with env variable.

**Comment by yongpingchen** [comment]
@satoshikumano please confirm 
1. path of config file can be set by env var `GWM_CONFIG`
2. path of db can be set in config file

File: main.go [hunk 1]
```diff
@@ -75,17 +86,19 @@ func main() {
 	if err != nil {
 		log.Fatalln("can't create bucket: ", err)
 	}
+
 	app := cli.NewApp()
 	app.Name = "gw-manager"
 	app.Version = "1.0.0"
-	app.Usage = "Sample app shows how to manage Gateway"
+	app.Usage = "Sample app shows how to manage Gateway. Specify the path of config file with env variable GWM_CONFIG" +
```
Comment (satoshikumano) [inline main.go]: People who are looking for the way of change the config file path might not find it.

Presumably, README is the better place to put the description.

**Comment by yongpingchen** [comment]
@satoshikumano 

> GWM_CONFIG_PATH or GWM_CONFIG_FILE_PATH might be clearer.

use `GWM_CONFIG_PATH`: 1169f03

> ./config.yml would be confusing when user set the custom path.
> Why do you omit the dbFile in the message?

Improve error message when failed to load config file and db file: 679ee00

File: main.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func main() {
 
 	db, err = bolt.Open(dbFile, 0600, nil)
 	if err != nil {
-		log.Fatalln("can't open db")
+		log.Fatalln("can't open " + dbFile)
```
Comment (satoshikumano) [inline main.go]: Why not include err here?

**Review by satoshikumano — APPROVED** [review]
LGTM

**Pull request merged**